packages/bpk-component-navigation-bar/src/BpkNavigationBar.js

```diff
@@ -29,7 +29,7 @@ const getClassNames = cssModules(STYLES);
 
 export type Props = {
   id: string,
-  title: string,
+  title: string | Element<any>,
```

Comment:
can we add a new test that renders with an element instead of a string?

Reply:
Done 👍